### ISSUE TITLE
Actions/Release Test and fine-tune the release

### ISSUE THREAD

**Issue by roblarsen** [h5bp/html5-boilerplate#2258]
I'm tired, so I'm just capturing a to-do list to vet the release process tomorrow. It works from end to end. 

- I created an annotated tag on the command line (See below, I forgot the  
- I pushed the tag.
- The action caught the new tag, zipped up the dist folder and attached it to a newly created GitHub release. 
- The action also published the package to npm. 

All of that worked.  I think I will need to update the formatting of the release annotation, but otherwise it's all pretty good. 

Open issues to clean up before Thursday/Monday (I'm aiming to release on Thursday, but will push it out to Monday if the blog post takes longer than I'm hoping.)

- [x]  Change the file name of the zip file to match previous versions. I forgot the `html5-boilerplate_` bit.
- [x] [Figure out how to zip the `dist` folder without including the folder itself.](https://github.com/h5bp/html5-boilerplate/blob/master/.github/workflows/publish.yaml#L19) Right now the folder contains the dist folder as a single child. We usually just have it be the _contents_ of the `dist` folder. My brain will hurt after this, I think. Maintaining a project like this means I have to learn about a lot of stuff. Learning the complexities of the `zip` command is not the most exciting stuff I've ever had to tackle. If anyone wants to explain this to me like I've five, I'm all ears. 
- [x] Test the [npm package](https://www.npmjs.com/package/html5-boilerplate) via all the different methods (including the new `npx` command)

![image](https://user-images.githubusercontent.com/361421/83588369-39c4f600-a51f-11ea-97c8-71e723ee031e.png)

**Comment by Jeroen-Matthijssens** [comment]
Instead of trying to figure out how to trick zip into dropping the top level directory you could use the `working-directory` directive on the step and zip everything.

```
- name: Create Zip Folder
  working-directory: dist
   run: zip -r ../${{ steps.get_version.outputs.VERSION }}.zip .
```

Just a thought.

**Comment by roblarsen** [comment]
@Jeroen-Matthijssens that is much more appealing

**Comment by Jeroen-Matthijssens** [comment]
@roblarsen np, I've been following the project for a long time, but only recently I have tried to actually participate and be helpful. Glad I could contribute in some way :)

**Comment by roblarsen** [comment]
closed via #2260
